consult.el

```diff
@@ -546,6 +561,11 @@ We use invalid characters outside the Unicode range.")
 
 ;;;; Miscellaneous helper functions
 
+(defun consult--replace-in-list (from to list)
+  "Replace occurrences of FROM with TO in LIST."
+  (mapcar (lambda (elt) (if (equal elt from) to elt))
+	  list))
```

Comment:
Inline this helper for now. If we need it more than once we can still extract it.